src/@types/process.ts

```diff
@@ -0,0 +1,9 @@
+export type Process = {
```

Comment:
I wasn't sure if this was the best place for this. Please let me know if I should move it somewhere else.